viz.Rmd

```diff
@@ -55,7 +64,7 @@ As with most coding tasks, it is totally fine (and quite common) to make mistake
 
 - **line plots** visualize trends with respect to an independent, ordered quantity (e.g. time)
 - **histograms** visualize the distribution of one quantitative variable (i.e., all its possible values and how often they occur) 
-- **scatter plots** visualize the distribution / relationship of two quantitative variables
+- **scatter plots** visualize the relationship between two quantitative variables
 - **bar plots** visualize comparisons of amounts
 
 All types of visualization have their (mis)uses, but three kinds are usually hard to understand or are easily replaced with an oft-better alternative.
```

Comment:
Edit: I think if we move the plots you created up to this section (see my comment below) then I don't think we need to move this. Since we can show the pictures then put this paragraph underneath the pictures